src/Microsoft.SqlTools.ResourceProvider.DefaultImpl/Microsoft.SqlTools.ResourceProvider.DefaultImpl.csproj

```diff
@@ -9,10 +9,10 @@
 		<OutputType>Library</OutputType>
 		<StartupObject />
 		<Description>Provides the default  for SqlTools applications.</Description>
-		<Copyright>� Microsoft Corporation. All rights reserved.</Copyright>
+		<Copyright>  Microsoft Corporation. All rights reserved.</Copyright>
```

Comment:
We have this defined in the root build.props (with the correct (C) symbol)  so it shouldn't be necessary here

```suggestion
```